projects/valtimo/document/src/lib/services/document.service.ts

```diff
@@ -306,6 +306,32 @@ export class DocumentService {
     );
   }
 
+  public findProcessDocumentDefinitionsForDocument(
+    documentId: string
+  ): Observable<ProcessDocumentDefinition[]> {
+    return this.http.get<ProcessDocumentDefinition[]>(
+      `${this.valtimoEndpointUri}v2/process-document/definition/document/${documentId}`
+    );
+  }
+
+  public findProcessDocumentDefinitionsForDocumentByStartableByUser(
+    documentId: string,
+    startableByUser: boolean
+  ): Observable<ProcessDocumentDefinition[]> {
+    return this.http.get<ProcessDocumentDefinition[]>(
+      `${this.valtimoEndpointUri}v2/process-document/definition/document/${documentId}?startableByUser=${startableByUser}`
+    );
+  }
+
+  public findProcessDocumentDefinitionsForDocumentByCanInitializeDocument(
+    documentId: string,
+    canInitializeDocument: boolean
+  ): Observable<ProcessDocumentDefinition[]> {
+    return this.http.get<ProcessDocumentDefinition[]>(
+      `${this.valtimoEndpointUri}v2/process-document/definition/document/${documentId}?canInitializeDocument=${canInitializeDocument}`
+    );
+  }
+
```

Comment:
These can all be merged into one with a search params object. Check logging-api.service.ts for an example